wiki/Skinning/Compendium/en.md

```diff
@@ -0,0 +1,261 @@
+[o!s]: /wiki/shared/mode/osu.png 
```

Comment:
add the title text
`[o!s]: /wiki/shared/mode/osu.png "osu!standard`

Reply:
Why? The links have the mode name and it get explained a few lines later

Reply:
it's easy to forget those explantations so title text might be good

Reply:
It'll do this
![deepinscreenshot_20180211093916](https://user-images.githubusercontent.com/14264992/36075114-80ec146c-0f0f-11e8-89d4-b70686646cc2.png)
this can help people know what icon it is since they are small icons and look similar to the others

Reply:
Ok. That makes sense. That will get changed.